src/components/v2v3/shared/SplitItem/SplitAmountValue.tsx

```diff
@@ -60,7 +60,9 @@ export function SplitAmountValue({ props }: { props: SplitProps }) {
         <span>
           (
           <CurrencySymbol currency={curr} />
-          {splitValueFormatted}
+          {props.showFees
+            ? splitValueFormatted
+            : formatWad(splitValue, { precision: 0 })}
```

Comment:
And would be great to define a `valueFormatted` `const` for this.

Reply:
Should we name it to `valueBeforeFeeFormatted` or smth like that?

Reply:
Agreeing with 'valueFormatted' here at the end! 🍻